TouchingMe.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+
+<html>
+     <head>
+
+     </head>
+   
+        <body>
+
+            <p> HAHAHAHAHAHA YOU TOUCHED ME LOSER HAHAHAHAHA</p>
+            
+            <script>
+            
+            const element = document.getElementById('grading-element-two');

```

Comment:
i ran `document.querySelector('#grading-element-two-consequence');` and got `null`

You're almost there. you need to `.appendChild(element)` to something